ui/src/labels/actions/index.ts

```diff
@@ -17,7 +17,12 @@ import {
 } from 'src/shared/copy/notifications'
 import {GetState} from 'src/types'
 
-export type Action = SetLabels | AddLabel | EditLabel | RemoveLabel
+export type Action =
+  | SetLabels
+  | AddLabel
+  | EditLabel
+  | RemoveLabel
+  | NotifyAction
```

Comment:
Aren't we using this Action to define what the reducer can accept? I'm afraid this reduces the type safety of the reducer. What would you think about removing line 25 and adding another type `ThunkActions = Action | NotifyAction` that we could use for dispatch?

Reply:
What I ultimately want to do is centralize all actions into a single format and location.  So, yes!